packages/assets/bin.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+import pathlib from 'path';
+import assetsFunc from './generate';
+
+const orgId = process.argv[2];
+const outputDirectory = process.argv[3];
+
+if (!orgId || !outputDirectory) {
+  console.error('orgId and outputDirectory required!');
+  process.exit(1);
+}
+
+const outputFolder = pathlib.join(process.cwd(), outputDirectory);
+
+try {
+  assetsFunc(orgId, outputFolder);
```

Comment:
Her kunne det også vært fint med litt oppdatering. F.eks

```suggestion
  console.log(`Writing assets for ${orgId} to ${outputFolder}`);
  await assetsFunc(orgId, outputFolder);
  console.log('Successfully written assets');
```

*NB* legg også merke til at jeg har lagt på en await her. Uten det vil ikke try/catch slå til. Gitt at vi bruker node versjon over 14.6 eller hva det er så skal top level await fungere her.

Reply:
Her bør det muligens være en await ja. Å ha den top-level medfører noen utfordringer knyttet til loopen som er omtalt her: https://stackoverflow.com/a/64655153

Får `SyntaxError: Cannot use import statement outside a module` før type er satt til module, og `TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /(......)/bin.ts` etter type er satt til module, samt at `target` og `module` i `tsconfig.json` er satt til `ESNext`.

`esmoduleinterop` løste heller ikke problemet, så skal se om jeg får komt meg rundt på et vis.

Reply:
Blir noe rot med versjoner og ts osv her ja. Kan unngå top-level await da, så unngår du problemet. Det er uansett mer "nice to have". Men alternativt vil dette sikkert fungere som forventet:

```
main();

async function main () {
  try {
    console.log(`Writing assets for ${orgId} to ${outputFolder}`);
    await assetsFunc(orgId, outputFolder);
    console.log('Successfully written assets');
  } catch (e) {
    console.error(e);
    process.exit(1);
  }  
} 
```

Reply:
Jepp, kommer fint rundt med noe slikt 👍